src/components/Notifications/NotificationItemImage.tsx

```diff
@@ -13,6 +13,7 @@ const NotificationItemImage = React.memo(
           horizontal: "left",
         }}
         badgeContent={badgeIcon}
+        isImage={typeof image === "string"}
```

Comment:
As this check is done in two places, would you mind creating a const variable named isImage that does this check? Also, would you mind adding to the comparison a `|| image instanceof String` that will check for strings created with the String object?